assets/src/components/ColorPaletteControl/ColorPaletteControl.js

```diff
@@ -1,17 +1,16 @@
 import { isEmpty } from 'lodash';
 import classnames from 'classnames';
 import { withInstanceId } from '@wordpress/compose';
-import { BaseControl } from '@wordpress/components';
-import { ColorPalette } from '@wordpress/components';
+import { BaseControl, ColorPalette } from '@wordpress/components';
 
 function ColorPaletteControl( { label, className, value, help, instanceId, onChange, options = [], ...passThroughProps } ) {
   const id = `inspector-color-palette-control-${ instanceId }`;
 
-  const optionsAsColors = options.map( ( { value, ...props } ) => ( { color: value, ...props } ) );
+  const optionsAsColors = options.map( ( { optionsValue, ...props } ) => ( { color: optionsValue, ...props } ) );
```

Comment:
I don't think this will still work 🤔 we probably need something like `options.map( ( { value: optionValue, ...props } ) => ( { color: optionValue, ...props } ) );`